client/src/partials/debtor_groups/debtor_groups.html

```diff
@@ -0,0 +1,289 @@
+                <td>
+                  <i ng-if="group.is_convention" class="glyphicon glyphicon-ok-sign"></i>
+                </td>
+                <td>
+                  <i ng-if="group.locked" class="glyphicon glyphicon-lock"></i>
+                </td>
+                <td>
+                  <a name="group-{{group.name}}" class="btn btn-xs btn-default"
+                    ng-click="DebtorGroupCtrl.update(group.uuid)">
+                    <i class="glyphicon glyphicon-pencil"></i> {{ 'FORM.EDIT' | translate }}
+                  </a>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+
+      <div class="col-md-6" ng-switch="DebtorGroupCtrl.view">
+
+        <div class="alert alert-info" ng-switch-default>
+          <h4>{{ "DEBTOR_GRP.TITLE" | translate }}</h4>
+          <p>{{ "DEBTOR_GRP.HELP_TXT_1" | translate }}</p>
+        </div>
+
+        <div ng-switch-when="action">
+          <form class="panel panel-primary" name="ActionForm" ng-submit="DebtorGroupCtrl.submit(ActionForm.$invalid)" novalidate>
+            <div class="panel-heading">
+              {{ DebtorGroupCtrl.actionTitle | translate }}
+            </div>
+            <div class="panel-body">
+
+              <!-- Debtor group info  -->
+              <fieldset>
+                <legend>{{ "COLUMNS.DEBTOR_GROUP" | translate }}</legend>
+
+                <div class="form-group" ng-class="{ 'has-error' : ActionForm.$submitted && ActionForm.name.$invalid }">
+                  <label class="control-label">
+                    {{ "COLUMNS.NAME" | translate }}
+                  </label>
+                  <input
+                    class="form-control"
+                    name="name"
+                    placeholder="{{ 'COLUMNS.NAME' | translate }}..."
+                    ng-model="DebtorGroupCtrl.debtorGroup.name"
+                    ng-change="DebtorGroupCtrl.refreshValidation()" required>
+                  <div class="help-block" ng-messages="ActionForm.name.$error" ng-show="ActionForm.$submitted">
+                    <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                  </div>
+                </div>
+
+                <!-- Find an account for debtor group -->
+                <div class="form-group" ng-class="{ 'has-error' : ActionForm.$submitted && ActionForm.account.$invalid }">
+                  <label class="control-label">
+                    {{ "COLUMNS.ACCOUNT" | translate }}
+                  </label>
+                  <input
+                    class="form-control"
+                    name="account"
+                    placeholder="{{ 'PLACEHOLDERS.ENTER.ACCOUNT' | translate }}..."
+                    uib-typeahead="account.id as account.account_number for account in DebtorGroupCtrl.accounts.data | filter:$viewValue | limitTo:8" class="form-control"
+                    typeahead-template-url="accounts.tpl"
```

Comment:
Could you move this template into `partials/templates`?  That way, when someone else looks for an account template in the future, they can just use yours!